### PR TITLE
Business View

### PR DESCRIPTION
**Pull request by cam-arch**
Added Image Carousel, added relative spacing and generally tidied the view up

### CODE REVIEW

File: src/components/BusinessView.tsx
Comment (bridges-wood) [inline src/components/BusinessView.tsx]: Why are there two styles here now?

File: src/components/ImagesCarousel.tsx
Comment (bridges-wood) [inline src/components/ImagesCarousel.tsx]: Move these outside of the component

File: src/components/ImagesCarousel.tsx
Comment (bridges-wood) [inline src/components/ImagesCarousel.tsx:35]: If we're going to use TouchableHighlight here, it would be good to do a similar thing with the Button component

File: src/components/ImagesCarousel.tsx
Comment (bridges-wood) [inline src/components/ImagesCarousel.tsx]: If you're referencing a local image, you need to use `require`.

File: src/components/RatingView.tsx
Comment (bridges-wood) [inline src/components/RatingView.tsx]: This style should be in the stylesheet, not the component.

**Review by bridges-wood — CHANGES_REQUESTED** [review]
Got a couple of changes/questions before approval

File: src/components/ImagesCarousel.tsx
Comment (cam-arch) [reply src/components/ImagesCarousel.tsx]: I didn't write that line.

File: src/components/ImagesCarousel.tsx
Comment (cam-arch) [reply src/components/ImagesCarousel.tsx:35]: OK

File: src/components/ImagesCarousel.tsx
Comment (cam-arch) [reply src/components/ImagesCarousel.tsx]: ok

File: src/components/RatingView.tsx
Comment (cam-arch) [reply src/components/RatingView.tsx]: I didn't put it in the stylesheet since it. was only 1 style change but I can if you want me to.

**Review by bridges-wood — APPROVED** [review]

**Pull request merged**
